### PR TITLE
Bonus style enforcement

### PR DESCRIPTION
**Pull request by Arch-Shaman**
Covers everything except fonts, because fonts are evil.

### CODE REVIEW

File: LuaUI/Configs/chickengui_config.lua [hunk 1]
```diff
@@ -14,22 +14,22 @@ configs.difficulties = {
...
-  chicken_tiamat = "\255\90\90\90",
+	chicken 				= "\255\184\100\255",
+	chicken_pigeon			= "\255\100\100\170",
+	chickena				= "\255\255\100\100",
+	chickens				= "\255\100\255\100",
+	--chicken_leaper		= "\255\255\120\80",
+	chickenwurm				= "\255\255\120\80",
+	chicken_sporeshooter 	= "\255\100\170\170",
+	--chicken_dodo   		= "\255\150\001\001",
+	chickenf				= "\255\255\255\100",
```
Comment (sprunk) [inline LuaUI/Configs/chickengui_config.lua:25]: Tabs in the middle are verboten. Keep spaces (ideally aligning the right side)

**Comment by GoogleFrog** [comment]
Conflicts,integral_menu_config_old was deleted about a week ago.

Also be wary of literal strings ([[...]]]). All the characters within them affect the text and they tend to indent with spaces for finer control chili-side.

File: LuaUI/Configs/chickengui_config.lua (hunk 1 above)
Comment (Arch-Shaman) [reply LuaUI/Configs/chickengui_config.lua:25]: Fixed. 3fd029b

File: LuaUI/i18nlib/i18n/plural.lua [hunk 2]
```diff
@@ -47,232 +49,192 @@ end
...
-          not between(n_100, 70, 79) and
-          not between(n_100, 90, 99)
-          and 'few') or
-         (n ~= 0 and n % 1000000 == 0 and 'many') or
-         'other'
+	if not isInteger(n) then
+		return 'other'
+	end
+	local n_10, n_100 = n % 10, n % 100
+	return (n_10 == 1 and not inside(n_100, {11,71,91}) and 'one') or (n_10 == 2 and not inside(n_100, {12,72,92}) and 'two') or (inside(n_10, {3,4,9}) and not between(n_100, 10, 19) and not between(n_100, 70, 79) and not between(n_100, 90, 99) and 'few') or (n ~= 0 and n % 1000000 == 0 and 'many') or 'other'
```
Comment (GoogleFrog) [inline LuaUI/i18nlib/i18n/plural.lua:96]: This became quite a long line. I prefer multi-line returns to this.

Comment (Arch-Shaman) [reply LuaUI/i18nlib/i18n/plural.lua]: Fixed 61dedd5

**Comment by GoogleFrog** [comment]
I am a little concerned with cross-project merge hell, but I suppose chili worked out.

**Comment by GoogleFrog** [comment]
@ArchShaman try updating the translation lib in chobby as well.

**Pull request merged**